### ISSUE TITLE
Jaws can't pick up balloon

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/balloons-and-static-electricity#532]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/689. Win 10 Chrome + Jaws. When using the screen reader, I can tab over to the balloon, and it will say press enter to pick it up. However, neither enter nor space will pick up the balloon.

**Visuals**

https://user-images.githubusercontent.com/41024075/130286065-c4fe35ef-4163-43ed-b819-60474283f0f1.MOV

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-dev.35/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-dev.35 2021-08-06 21:46:41 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36
Language: en-US
Window: 1280x657
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe - I am pretty sure this is the same problem as https://github.com/phetsims/john-travoltage/issues/442 and I expect it to be fixed in master. Could you please confirm?

If fixed in master would you like a new dev version to finish testing JAWS?

**Comment by KatieWoe** [comment]
Balloon is pick-up-able in master. I'll leave it up to @jbphet if he wants a new dev for it. I'm about to leave so I can't do it until Monday.

**Comment by jbphet** [comment]
> I'll leave it up to @jbphet if he wants a new dev for it.

Let's verify it in the first RC.  I'll make sure it's on the issues-to-verify list in the RC issue.

**Comment by KatieWoe** [comment]
This seems fixed in rc1
